src/backend/gl/src/queue.rs

```diff
@@ -472,8 +472,45 @@ impl CommandQueue {
             com::Command::BindProgram(program) => unsafe {
                 self.share.context.UseProgram(program);
             }
-            com::Command::BindBlendSlot(slot, blend) => {
-                state::bind_blend_slot(&self.share.context, slot, &blend);
+            com::Command::BindBlendSlot(slot, ref blend) => {
+                state::bind_blend_slot(&self.share.context, slot, blend);
+            }
+            com::Command::BindAttribute(ref attribute, handle, stride, function_type) => unsafe {
+                use native::VertexAttribFunction::*;
+
+                let &native::AttributeDesc { location, size, format, offset, .. } = attribute;
+                let offset = offset as *const gl::types::GLvoid;
+                let gl = &self.share.context;
+
+                gl.BindBuffer(gl::ARRAY_BUFFER, handle);
+
+                match function_type {
+                    Float => gl.VertexAttribPointer(location, size, format, gl::FALSE, stride, offset),
```

Comment:
no support for normalized values?

Reply:
Not yet, I'll try to add a follow-up PR soon